### PR TITLE
fix(DB/Loot) Droprate of Doctor's Key from Doctor Gutrick

### PR DESCRIPTION
**Pull request by Tereneckla**
## Changes Proposed:
-  Change droprate of The Doctor's Key to 100% on Doctor Gutrick 
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/15034
- Closes https://github.com/chromiecraft/chromiecraft/issues/5032

## SOURCE:

https://www.wowhead.com/npc=21778/doctor-gutrick#drops

seems like a wowhead sampling error that its not 100% as the horde version is 100%. there are also no comments suggesting it didnt drop

## Tests Performed:

- 
- 

## How to Test the Changes:

1. .go c 86719
2. .damage 100000
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Gultask** [comment]
> Missing Semicolon (;) or multiple newlines at the end of the file.
Oh boy this error :P
Try adding `--` at the top of the file.

**Comment by Gultask** [comment]
image

**Comment by Gultask** [comment]
image

**Review by Nyeriah — APPROVED** [review]

**Comment by Nyeriah** [comment]
Does he drop nothing else? Currently query deletes everything but the key

**Comment by Gultask** [comment]
Would be best to return to the original iteration of this PR:
```
UPDATE `creature_loot_template` SET `Chance` = 100 WHERE (`Entry` = 21778) AND (`Item` = 30712);
```

**Comment by Tereneckla** [comment]
He did not have any other drops before and no other drops are recorded on wowhead

File: data/sql/updates/pending_db_world/rev_1676755297977733200.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `creature_loot_template` SET `Chance` = 100 WHERE (`Entry` = 21778) AND (`Item` = 30712);
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1676755297977733200.sql:1]:
```suggestion
UPDATE `creature_loot_template` SET `Chance` = 100 WHERE (`Entry` = 21778) AND (`Item` = 30712);

```

**Pull request merged**